projects/www/src/app/pages/guide/signals/rxjs-integration.md

```diff
@@ -6,7 +6,7 @@ RxJS is still a major part of NgRx and the Angular ecosystem, and the `@ngrx/sig
 
 The `rxMethod` is a standalone factory function designed for managing side effects by utilizing RxJS APIs.
 It takes a chain of RxJS operators as input and returns a reactive method.
-The reactive method can accept a static value, signal, or observable as an input argument.
+The reactive method can accept a static value, a reactive computation (like a Signal), or observable as an input argument.
```

Comment:
I am using the term "reactive computation" instead of auto-tracking functions because that's what the Angular team calls it in `linkedSignal`.